### PR TITLE
Display warning if name already used

### PR DESCRIPTION
**Pull request by kookiCT**
Requires dev_preprocess_temp02 first : https://github.com/rodeofx/tk-nuke-writenode/pull/12

### CODE REVIEW

File: python/tk_nuke_writenode/handler.py [hunk 1]
```diff
@@ -812,6 +812,41 @@ def __get_publish_template(self, node, is_proxy=False):
...
+        template = self.__get_render_template(node)
+        use_version = bool(template.keys.get("version"))
+
+        if not use_version:
+            output_key_name = None
+            for key_name in ["output", "channel"]:
+                if template.keys.get(key_name):
+                    output_key_name = key_name
+
+            fields = {}
```
Comment (foutoucour) [inline python/tk_nuke_writenode/handler.py:839]: look like you have some duplicated code here with #12 , do you think you could factorize it?

Comment (kookiCT) [inline python/tk_nuke_writenode/handler.py:839]: This is the factorization of PR #12... which is why the PR goes to the dev_preprocess_temp02 branch for now ;)

Comment (foutoucour) [inline python/tk_nuke_writenode/handler.py:839]: ok ;)

**Comment by foutoucour** [comment]
:+1:

File: python/tk_nuke_writenode/handler.py [hunk 2]
```diff
@@ -1253,6 +1267,24 @@ def __set_output(self, node, output_name):
         """
         self._app.log_debug("Changing the output for node '%s' to: %s" % (node.name(), output_name))
         
+        # check if new output_name is not already used
+        name_warning = ""
+        used_output_names = self.__get_used_output_names(node)
+        if output_name in used_output_names:
+            name_warning = "<br>".join(self.__wrap_text(
+                        "This output name is already used ! "
```
Comment (mfe) [inline python/tk_nuke_writenode/handler.py]: Wrong indentation.

**Comment by mfe** [comment]
Except indentation couac, :+1:

File: python/tk_nuke_writenode/handler.py (hunk 2 above)
Comment (kookiCT) [inline python/tk_nuke_writenode/handler.py]: Fixed !

File: python/tk_nuke_writenode/handler.py [hunk 3]
```diff
@@ -812,6 +812,41 @@ def __get_publish_template(self, node, is_proxy=False):
         else:
             return self.__get_template(node, "publish_template")
     
+    def __get_used_output_names(self, node):
```
Comment (JeanChristopheMorinRodeoFX) [inline python/tk_nuke_writenode/handler.py:815]: docstr?

Comment (kookiCT) [inline python/tk_nuke_writenode/handler.py:815]: Done

**Comment by kookiCT** [comment]
Closing in favor of #15 that goes into master_rdo

**Pull request closed without merging**